utils/constants.ts

```
@@ -24,6 +24,7 @@ export const CRM_ENDPOINTS = {
 export const ORG_AND_EMP_URL = process.env.ORG_AND_EMP_URL;
 export const OKR_URL = process.env.OKR_URL;
 export const PAYROLL_URL = process.env.PAYROLL_URL;
+export const CRM_URL = process.env.CRM_URL;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'CRM_URL'. Consider to delete it or rename it.

'CRM_URL' is defined here:


(lint/suspicious/noRedeclare)